### ISSUE TITLE
Time Limit lock screen issue

### ISSUE THREAD

**Issue by easpro** [rcaelers/workrave#324]
When time limit is reached, the Workrave lock screen notice appears.  If I press alt, ctrl, del to sign out, then log back on, the lock screen does not appear.  PC is accessible again.

**Comment by rcaelers** [comment]
Indeed. But when you log back in, the break is overdue. So I expect you get a new break reminder window in a few minutes (depending on postpone duration). Does this indeed happen? What behavior would you expect?

**Comment by easpro** [comment]
Postpone and skip were not checked.

**Comment by Chealer** [comment]
@easpro: Please see [that checklist](https://blog.michael.gr/post/2014-11-bug-reporting-checklist/), in particular sections 6 and 8 to 12. This is an ITS, so all tickets are about issues; the title you choose does not need to indicate "issue", but should focus on describing what distinguishes this issue.

If by "lock screen" you mean the rest break screen, I suggest retitling to:
> Logging out/in postpones rest break

[🅭🄍](https://www.philippecloutier.com/Common+infrastructure+licensing#its)
